postman/config.yml

```diff
@@ -5,12 +5,16 @@ name: postman
 title: Postman
 
 # Long-form description of the quickstart (required)
-description: |
-  Observe the performance of your APIs using Postman and New Relic
+description:|
+  ## Postman's New Relic Integration
+
+  Postman is an API platform that allows you to manage your APIs throught it's lifecycle. Postman's
+  integration with NewRelic allows you to observe the performance of your APIs by sending results from Postman monitor
+  run to NewRelic and getting key API performance information back in Postman's API platform
```

Comment:
I'd suggest the following:

Postman is an API platform for building and using APIs. Postman simplifies each step of the API lifecycle and streamlines collaboration so you can create better APIs—faster. Postman's integration with New Relic allows you to observe the performance of your APIs by sending results from Postman Monitor to New Relic and returning critical API performance data back ti Postman's API platform.